core/services/feeds/models.go

```diff
@@ -39,9 +39,12 @@ const (
 )
 
 type JobProposal struct {
-	ID             int64
-	Spec           string
-	Status         JobProposalStatus
+	ID int64
+	// UUID is the unique id of the proposal in FMS.
```

Comment:
Should we call it FeedManagerUUID as its actually a foreign ID? Just UUID makes me think that its a node generated UUID

Reply:
Renamed to `RemoteUUID` because FeedManagerUUID implies its the Feed Manager's UUID and not the job itself